ipfs-cids/.moonbeam-cid

```diff
@@ -0,0 +1 @@
+QmZ97sLQtub7saRVL5fQzpVTo2MzU56j4jBe9JnMzL1iPg
```

Comment:
why is it different from other https://github.com/nova-wallet/subquery-nova/pull/174/files#diff-f0abcb7585561dba99231d6f33a8dc519dc467a6b3699f21c49e54c4232cc093

Reply:
It's like UUID for IPFS deploy process, more info here:
https://academy.subquery.network/tr/run_publish/publish.html#publish-your-subquery-project-to-ipfs